### PR TITLE
docs: Added link of contributing guidelines to README

### LINKED ISSUE #58: docs: Add link in README
### Is your feature request related to a problem? Please describe.
The Contributing section of the README.md doesn't have the link in the "contributing guidelines"

### Describe the solution you'd like
Add the link of AnitaB.org contributing guidelines over there

### Additional context
This is required for any newcomer to first read the contributing guidelines before making any contribution.

### PR DESCRIPTION
**Pull request by khushishikhu**
### Description
Added link of contributing guidelines

Fixes #58 

### Type of Change
- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged

### CODE REVIEW

File: README.md
Comment (codesankalp) [inline README.md]: Please Correct the spelling of `Contributing`

**Review by codesankalp — CHANGES_REQUESTED** [review]
@khushishikhu You have also included the commit of this PR - https://github.com/anitab-org/open-source-programs-backend/pull/52
Please remove them as that commit is not needed here because it is only for solving the issue related to link for contributing guidelines and i am seeing the changes of setup also here.
Please correct it. :smiley:

**Comment by isabelcosta** [comment]
@khushishikhu you can fix the merge conflicts now 🎉

File: README.md
Comment (isabelcosta) [inline README.md]:
```suggestion
Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/open-source-programs-backend/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/open-source-programs-backend/blob/develop/REPORTING_GUIDELINES.md)
```

**Review by isabelcosta — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Can you please remove all the changes that are not related to the sole link you had to change as part of the issue? @khushishikhu 

It's good practice to keep the changes specific to the issue you are solving.

**Comment by khushishikhu** [comment]
@isabelcosta  I have commit the changes. please have a look now

**Comment by codesankalp** [comment]
@khushishikhu Please remove the previous commits or update your `patch-2` branch with this repo develop branch.

**Comment by isabelcosta** [comment]
@khushishikhu could you please remove the changes that are in the file, but irrelevant to the issue :) I think these are from another issue

File: README.md
Comment (Rahulm2310) [inline README.md]: @khushishikhu Can you please make paths to reporting guidelines & code of conduct also relative similar to contributing guidelines.

Comment (khushishikhu) [reply README.md]: I don't think that it would be same as it is in the contributing guidelines

> @khushishikhu Can you please make paths to reporting guidelines & code of conduct also relative similar to contributing guidelines.

Comment (Amulya-coder) [inline README.md]:
```suggestion
Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](CODE_OF_CONDUCT.md) and [Reporting Guidelines](REPORTING_GUIDELINES.md)
```
@khushishikhu the relative paths help to make this link accessible in multiple forks, GitHub will automatically render the link to the specified path and I observed the path will also become shorter. I also come to know about that yesterday when I was reviewing the pr by @isabelcosta 🤗

**Review by Amulya-coder — CHANGES_REQUESTED** [review]

**Review by isabelcosta — DISMISSED** [review]
Awesome 👏🏾 @khushishikhu 
Thank you so much for this contribution!

**Comment by isabelcosta** [comment]
Can I get one more approval please @codesankalp @Amulya-coder @Rahulm2310 ?

**Review by Amulya-coder — DISMISSED** [review]
LGTM!! @khushishikhu 🎉.

File: README.md
Comment (codesankalp) [inline README.md]: why these `=====` here?

File: README.md
Comment (codesankalp) [inline README.md]: `Y` must be capital.

**Review by codesankalp — CHANGES_REQUESTED** [review]
I see some `=====` in readme that is due to the previous commits merging.
I think this needs to be changed.

**Comment by isabelcosta** [comment]
I am so confused, I approved, when I saw that there was only the link change 🙈

File: README.md
Comment (isabelcosta) [inline README.md]:
```suggestion
```

File: README.md
Comment (isabelcosta) [inline README.md]:
```suggestion
```

File: README.md
Comment (isabelcosta) [inline README.md]:
```suggestion
```

File: README.md
Comment (isabelcosta) [inline README.md]:
```suggestion
```

File: README.md
Comment (isabelcosta) [inline README.md]:
```suggestion
```

**Review by isabelcosta — APPROVED** [review]
Now it looks appropriate to the issue content :) I changed since it was some small changes to documentation.
Thank you for working on this @khushishikhu

**Comment by isabelcosta** [comment]
@codesankalp can you take a look again?

**Review by codesankalp — APPROVED** [review]
Now it looks good, :+1: :tada: .
It can be merged now.
@isabelcosta

**Pull request merged**
